src/librustc/diagnostics.rs

```diff
@@ -1383,38 +1383,6 @@ struct Foo<T: 'static> {
 ```
 "##,
 
-E0312: r##"
```

Comment:
The error E0312 still exists so why not writing its long error diagnostic instead of just removing the old one?

Reply:
At it. Just thought you should know.

Reply:
done